Week4/Blockbuster/app/src/main/java/com/example/blockbuster/data/DataManager.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.blockbuster.data
```

Comment:
Really nice view model class. I like how you've organized this into concise, short functions, very readable.